pages/faq.md

```diff
@@ -113,7 +113,7 @@ The `getfacl` command is an easy way to see the permissions of a file or directo
 
 During installation you may see an error such as `"conda.exe: error while loading shared libraries: libz.so.1: failed to map segment from shared object: Operation not permitted"`  
 
-This relates to an issue storing temporary files created during installation. Please create a temporary directory within your project or Panasas scratch directory and specify that location in the installation command. For example:
+This relates to an issue storing temporary files created during installation. Please create a temporary directory within your project or global scratch directory and specify that location in the installation command. For example:
 ```
 mkdir /projects/academic/<group_name>/condatemp
 TMPDIR=/projects/academic/<group_name>/condatemp ./Anaconda3-2020.02-Linux-x86_64.sh --prefix=/projects/academic/<group_name>/<install_dir>
```

Comment:
TMPDIR=/projects/academic/[YourGroupName]/condatemp ./Anaconda3-2020.02-Linux-x86_64.sh --prefix=/projects/academic/[YourGroupName]/[InstallDir]

We haven't discussed arbitrary software paths yet. I think for now either [InstallDir] or <install_dir> format is fine.